File: support_tools/swigwin-3.0.12/Examples/test-suite/ruby/grouping_runme.rb
#!/usr/bin/env ruby
#
# Put description here
#
# 
# 
# 
#

require 'swig_assert'

require 'grouping'

x = Grouping.test1(42)
if x != 42
  raise RuntimeError
end

Grouping.test2(42)

x = Grouping.do_unary(37, Grouping::NEGATE)
if x != -37
  raise RuntimeError
end

Grouping.test3 = 42

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/grouping_runme.py
import grouping

x = grouping.test1(42)
if x != 42:
    raise RuntimeError

grouping.test2(42)

x = grouping.do_unary(37, grouping.NEGATE)
if x != -37:
    raise RuntimeError

grouping.cvar.test3 = 42
grouping.test3 = 42
